infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackController.kt

```diff
@@ -225,7 +225,7 @@ class LocationTrackController(
     }
 
     @PreAuthorize(AUTH_ALL_READ)
-    @GetMapping("/track-numbers/{publicationState}/{trackNumberId}/location-tracks")
+    @GetMapping("/track-numbers/{publicationState}/location-tracks/{trackNumberId}")
```

Comment:
En kyllä keksi parempaakaan järjestystä, mutta hieman weird nyt kun track numberin id on location-tracksin alla

Reply:
Olisko parempi jos tuo polku olisikin `.../location-tracks-of/...`?